integration-test/src/test/java/org/apache/iotdb/db/it/IOTDBLoadTsFileIT.java

```diff
@@ -0,0 +1,579 @@
+        Statement statement = connection.createStatement()) {
+
+      statement.execute(String.format("load \"%s\" sglevel=2", tmpDir.getAbsolutePath()));
+
+      try (ResultSet resultSet =
+          statement.executeQuery("select count(*) from root.** group by level=1,2")) {
+        if (resultSet.next()) {
+          long sg1Count = resultSet.getLong("count(root.sg.test_0.*.*)");
+          Assert.assertEquals(writtenPoint1, sg1Count);
+          long sg2Count = resultSet.getLong("count(root.sg.test_1.*.*)");
+          Assert.assertEquals(writtenPoint2, sg2Count);
+        } else {
+          Assert.fail("This ResultSet is empty.");
+        }
+      }
+    }
+  }
+
+  private static class SchemaConfig {
+    private static final String STORAGE_GROUP_0 = "root.sg.test_0";
+    private static final String STORAGE_GROUP_1 = "root.sg.test_1";
+
+    // device 0, nonaligned, sg 0
+    private static final String DEVICE_0 = "root.sg.test_0.d_0";
+    private static final MeasurementSchema MEASUREMENT_00 =
+        new MeasurementSchema("sensor_00", TSDataType.INT32, TSEncoding.RLE);
+    private static final MeasurementSchema MEASUREMENT_01 =
+        new MeasurementSchema("sensor_01", TSDataType.INT64, TSEncoding.RLE);
+    private static final MeasurementSchema MEASUREMENT_02 =
+        new MeasurementSchema("sensor_02", TSDataType.DOUBLE, TSEncoding.GORILLA);
+    private static final MeasurementSchema MEASUREMENT_03 =
+        new MeasurementSchema("sensor_03", TSDataType.TEXT, TSEncoding.PLAIN);
+
+    // device 1, aligned, sg 0
+    private static final String DEVICE_1 = "root.sg.test_0.a_1";
+    private static final MeasurementSchema MEASUREMENT_10 =
+        new MeasurementSchema("sensor_10", TSDataType.INT32, TSEncoding.RLE);
+    private static final MeasurementSchema MEASUREMENT_11 =
+        new MeasurementSchema("sensor_11", TSDataType.INT64, TSEncoding.RLE);
+    private static final MeasurementSchema MEASUREMENT_12 =
+        new MeasurementSchema("sensor_12", TSDataType.DOUBLE, TSEncoding.GORILLA);
+    private static final MeasurementSchema MEASUREMENT_13 =
+        new MeasurementSchema("sensor_13", TSDataType.TEXT, TSEncoding.PLAIN);
+
+    // device 2, non aligned, sg 1
+    private static final String DEVICE_2 = "root.sg.test_1.d_2";
+    private static final MeasurementSchema MEASUREMENT_20 =
+        new MeasurementSchema("sensor_20", TSDataType.INT32, TSEncoding.RLE);
+
+    // device 3, non aligned, sg 1
+    private static final String DEVICE_3 = "root.sg.test_1.d_3";
+    private static final MeasurementSchema MEASUREMENT_30 =
+        new MeasurementSchema("sensor_30", TSDataType.INT32, TSEncoding.RLE);
+
+    // device 4, aligned, sg 1
+    private static final String DEVICE_4 = "root.sg.test_1.a_4";
+    private static final MeasurementSchema MEASUREMENT_40 =
+        new MeasurementSchema("sensor_40", TSDataType.INT32, TSEncoding.RLE);
+  }
+
+  public class TsFileGenerator implements AutoCloseable {
```

Comment:
It seems that `TsFileGenerator` is a common tool in integration test, despite to depend on those internal classes of tsFile directly, you can
1. Move it to src/main, as a separate class in a package.
2. Refine the methods of this class, let the parameters are raw Java types(int, string, not Path etc).